build/windows.yml

```diff
@@ -3,15 +3,15 @@
 parameters:
   target: ''
   rust_version: 'stable'
+  repo: 'BurntSushi/ripgrep'
```

Comment:
I don't want to have this same thing duplicated in a bunch of places. It can be different for mac but there should only be one place to update for the others

Reply:
updated, thanks!